helm/featurehub/templates/mr-deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "featurehub.managementRepository.name" . }}
+  labels:
+    {{- include "featurehub.managementRepository.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.managementRepository.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "featurehub.managementRepository.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "featurehub.managementRepository.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "featurehub.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ include "featurehub.managementRepository.name" . }}
```

Comment:
Can you define memory and cpu limits?

Reply:
there is a resources section to include from the yaml, but its overall, so it should probably be per app

Reply:
i.e. they all get the same resource allocated if its defined